src/sequence/cache/index.js

```diff
@@ -0,0 +1,25 @@
+const { isFunction } = require("lodash");
+
+const defaultStrategy = require("./strategies/default");
+const asyncStrategy = require("./strategies/async");
+
+
+let cacheStrategy = defaultStrategy();
```

Comment:
Would there ever be a scenario for having two rapscallion instances which would have differing cache strategies?

You've got a global mutable singleton, which ties your hands a bit vs. maybe creating a cache object to initialize and pass to renderers...